universal-login-sdk/test/e2e/DeployedWallet.ts

```diff
@@ -42,9 +42,9 @@ describe('E2E: DeployedWallet', async () => {
     });
 
     it('returns the correct number of required signatures after update', async function () {
-      let {waitToBeSuccess} = await deployedWallet.addKey(otherWallet.address, {gasToken: mockToken.address});
+      let {waitToBeSuccess} = await deployedWallet.addKey(otherWallet.address, {...DEFAULT_PAYMENT_OPTIONS, gasToken: mockToken.address});
```

Comment:
Make `PaymentOptions` have optional keys so you can specify only some subset of parameters

Reply:
Done